program/cylindrically-rotate-a-matrix-by-90-degrees/rotateMatrix.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+let N = 3;
+
+function rotate90Clockwise(arr) {
+  for (j = 0; j < N; j++) {
+    for (i = N - 1; i >= 0; i--)
+      console.log(arr[i][j] + " ");
+  }
+}
+
+
+let arr = [[1, 1, 1], [2, 2, 2], [3, 3, 3]];
+rotate90Clockwise(arr);
```

Comment:
This code won't work in a strict mode . 
Either you need to remove 'using strict' or fix the variable j declaration